pkg/kubelet/cm/dra/plugin/plugin.go

```diff
@@ -53,9 +100,11 @@ func (h *RegistrationHandler) RegisterPlugin(pluginName string, endpoint string,
 
 	// Storing endpoint of newly registered DRA Plugin into the map, where plugin name will be the key
 	// all other DRA components will be able to get the actual socket of DRA plugins by its name.
-	draPlugins.Set(pluginName, &Plugin{
+	// By default we assume the supported plugin version is v1alpha3
+	draPlugins.add(pluginName, &plugin{
 		conn:                    nil,
 		endpoint:                endpoint,
+		version:                 v1alpha3Version,
```

Comment:
When / how does it ever get set to v1alpha2Version?

Reply:
as soon as the first call for v1alpha3 fails like [here](https://github.com/kubernetes/kubernetes/blob/c3139769f8ba4df3674da4d913af1aaece873a47/pkg/kubelet/cm/dra/plugin/client.go#L159)